plugins/chat/app/models/chat/channel.rb

```diff
@@ -20,6 +23,10 @@ class Channel < ActiveRecord::Base
              foreign_key: :chat_channel_id
     has_many :threads, class_name: "Chat::Thread", foreign_key: :channel_id
     has_one :chat_channel_archive, class_name: "Chat::ChannelArchive", foreign_key: :chat_channel_id
+    belongs_to :last_message,
```

Comment:
Why `belongs_to` ?

Reply:
Because the foreign key to the message is stored on this model, same reason we use belongs_to for channel/original_message/original_message_user on threads and chatable here on channel

Reply:
That always sounds super backward to me, but ok :+1:

Reply:
Yes me too, I've never liked it. The way I remember it is that, in Toy Story Woody has Andy's name written on his shoe because he belongs to Andy, whereas Andy has one Woody but also has one Buzz.